content/docs/ui/sending-email/getting-started-with-automation.md

```diff
@@ -435,19 +435,17 @@ The Automation beta includes the option to create a one-time email campaign, sim
 
 ## Migrating from Marketing Campaigns
 
-### Migrating Campaign HTML
+### Migrating Marketing Campaigns HTML
 
-*To migrate HTML from a Marketing Campaigns template or email:*
+*To migrate HTML from a Marketing Campaigns template into a new email:*
 
 1. Navigate to [Marketing Campaigns](https://sendgrid.com/marketing_campaigns/ui/campaigns) and locate the campaign you wish to migrate.
-1. Hover over the action menu until the icons appear.
-1. Select the **Export HTML** icon.
-   The campaign HTML downloads to the folder specified on your computer
+1. Hover over the action menu until the icons appear and select the **Export HTML** icon.
+   The campaign HTML downloads to the folder specified on your computer.
 1. Navigate to the Automation dashboard.
-1. Locate the automation you want to add the exported email to and hover over the action menu.
-1. Select **Edit**.
-1. Within the automation, find the email you want to add the content to and click **Edit**.
-![]({{root_url}}/img/edit-email-automation.png "Edit email content")
+1. Locate the Automation you want to add the exported email to and hover over the action menu.
+1. Select **Add Email Content**.
+![]({{root_url}}/img/add_email_content.png "Add email content")
 8. Select **Blank Template**.
 9. Select the code editor.
 10. Paste the HTML you copied from the downloaded file into the code editor and then click **Preview**.
```

Comment:
We're thinking of adding in another section for importing design HTML into already existing automation emails since the steps are a little different, what do you think? (We also think it could be fine without for now)

Reply:
Also, I'm considering maybe adding in a TIP about how you could also use the "Import Custom Drag & Drop HTML" under Build and Advanced in Design Editor underneath (line 451) "10. Paste the HTML you copied from the downloaded file into the code editor and then click **Preview**." under "Migrating Marketing Campaigns HTML".

Reply:
I think that could work. I'd say that last time that I checked, usage of the Design vs. Code editor was about 50/50. So, there will be a lot of customers exporting templates from Design editor and needing to reimport into Design. Guess the tip (vs calling out both as options in the doc steps itself) makes it feel like Design is less important/used? Just want to be sure we don't miss out on the clarity and end up causing noise for support (especially since it's a key piece of customers being able to migrate over to the new MC).

Let me know your thoughts!

Reply:
Alright, I've gone ahead and made a new section for it for now then!

Let me know if everything looks correct!

Reply:
(Once I manage to push my changes properly via the terminal, haha)